src/omv/imlib/framebuffer.h

```diff
@@ -33,7 +33,9 @@ typedef struct jpegbuffer {
     int32_t enabled;
     int32_t quality;
     mutex_t lock;
-    uint8_t pixels[];
+    int64_t __padding;
```

Comment:
Why is this padding required ?

Reply:
Just to ensure the 32-byte alignment. The align macro also does this too. I wanted to show that 32-bytes of the structure were in use. But, it's unnecessary.